ChangeLog.md

```diff
@@ -3,6 +3,10 @@
 ## Next
 * 
 
+## Version 1.17.0
+* Server: Upgrade hyper to 0.14.28
+* Libs/Java: Use Java time instead of threetenbp
```

Comment:
Highlight that this is potentially breaking (depending on how you use the lib), but was a necessary change to make things much simpler for people.